### DISCUSSION TITLE
py-script src argument makes pyscript fail to load anything

### DISCUSSION THREAD

**Discussion by XLuma** [pyscript/pyscript#1286] [Q&A]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

```
`<html>
    <head>
      <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
      <script defer src="https://pyscript.net/latest/pyscript.js"></script>
    </head>

  <body>
    <py-script src="example.py"></py-script>
  </body>
</html>`
```

adding the src argument to py-script tag causes pyscript to not execute code in local modules imported with the src tag. Manually writing code inside <py-script> tags work normally though

This wasn't happening until today. Not exactly sure what is happening

### Console info

```shell
`Traceback (most recent call last):
  File "/home/pyodide/pyscript/__init__.py", line 484, in uses_top_level_await
    return TopLevelAsyncFinder().is_source_top_level_await(source)
  File "/home/pyodide/pyscript/__init__.py", line 466, in is_source_top_level_await
    node = ast.parse(source)
  File "/lib/python3.10/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    <html>
    ^
SyntaxError: invalid syntax`
```

### Additional Context

using npm to fire up a web server

**Comment by XLuma** [comment]
Edit: Checked the networking tab in dev tools from Firefox: it seems example.py, when fetched, somehow fetches index.html instead.

**Comment by FabioRosado** [comment]
Thank you for opening this issue with us @XLuma, I'm turning this issue into a discussion 😄 

 can you provide your code? From the traceback its hard to see what's happening

**Reply by XLuma** [reply]
[test-pyscript.zip](https://github.com/pyscript/pyscript/files/10972453/test-pyscript.zip)
here is a zip file containing the index.html and example.py referenced in the issue !

**Reply by marimeireles** [reply]
I can maybe also help but I'd rather someone to paste the code in some service like pastebin.com
Thanks

**Comment by XLuma** [comment]
I did some more testing: it seems the issue arises when using Node to host a web server. Tested using python http.server, which works perfectly fine to start a web server, and referenced scripts will work just fine.
One issue though is that scripts inside child folders will not import properly, unsure if this is a limitation of web ?

**Comment by MrPimples** [comment]
I got same error with the simple code like in the file
[index.zip](https://github.com/pyscript/pyscript/files/11010543/index.zip)

**Reply by FabioRosado** [reply]
Just opened up your file, it seems your python code is added in the same line. Python uses indentation so your code will not run. You will have to change it like:

```html
<py-script>
    arr = [1,2,3,4]
    for i in arr: 
        print('Test' + i)
</py-script>
```
